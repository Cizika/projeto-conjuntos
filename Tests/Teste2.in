1 1 10
1 2 5
1 2 3
1 3 10
3 10
2 3 10
2 1 10
3 10
4 1 3
1 1 3
5 1 2
0