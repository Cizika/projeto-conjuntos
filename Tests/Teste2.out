--------------------------------------------------
CONJUNTOS MATEMÁTICOS
1 - Insere um elemento no conjunto desejado
2 - Remove um elemento do conjunto desejado
3 - Verifica se o elemento pertence aos conjuntos
4 - União entre conjuntos
5 - Intersecção entre conjuntos
0 - Finalizar Programa
--------------------------------------------------
Conjunto 1: {}
Conjunto 2: {}
Conjunto 3: {}
--------------------------------------------------
Conjunto 1: {10 }
Conjunto 2: {}
Conjunto 3: {}
--------------------------------------------------
Conjunto 1: {10 }
Conjunto 2: {5 }
Conjunto 3: {}
--------------------------------------------------
Conjunto 1: {10 }
Conjunto 2: {3 5 }
Conjunto 3: {}
--------------------------------------------------
Conjunto 1: {10 }
Conjunto 2: {3 5 }
Conjunto 3: {10 }
--------------------------------------------------
O elemento 10 pertence ao conjunto 1
O elemento 10 NÃO pertence ao conjunto 2
O elemento 10 pertence ao conjunto 3
--------------------------------------------------
Conjunto 1: {10 }
Conjunto 2: {3 5 }
Conjunto 3: {}
--------------------------------------------------
Conjunto 1: {}
Conjunto 2: {3 5 }
Conjunto 3: {}
--------------------------------------------------
O elemento 10 NÃO pertence ao conjunto 1
O elemento 10 NÃO pertence ao conjunto 2
O elemento 10 NÃO pertence ao conjunto 3
--------------------------------------------------
União(1, 3): {}
--------------------------------------------------
Conjunto 1: {3 }
Conjunto 2: {3 5 }
Conjunto 3: {}
--------------------------------------------------
Intersecção(1, 2): {3 }
--------------------------------------------------
Finalizando Programa
